nvflare/app_common/workflows/model_controller.py

```diff
@@ -36,7 +37,7 @@
 
 
 @experimental
-class ModelController(Controller, FLComponentWrapper):
+class ModelController(Controller, FLComponentWrapper, WFControllerSpec, ABC):
```

Comment:
maybe we move this "WFControllerSpec, ABC" outside of here?

Reply:
decided to keep since `ModelController` uses `run()`